website/src/documentation/programming-guide.md

```diff
@@ -39,6 +39,9 @@ how to implement Beam concepts in your pipelines.
   </ul>
 </nav>
 
+{:.language-py}
+The Python SDK supports Python 2.7, 3.5, or higher; soon, new Python SDK releases won't support Python 2.7 ([BEAM-8371](https://issues.apache.org/jira/browse/BEAM-8371)). For best results, use Beam with Python 3.
```

Comment:
How about, instead of `new Python SDK releases won't support` -> `starting from 2020 newer Python SDKreleases won't support` ?

Reply:
Based on latest round of discussions in the mailing list, we decided that we will do it somewhere in 2020, but not necessarily in the beginning. We will revisit this conversation. For now I think we can say:

New Python SDK releases will stop supporting Python 2.7 in 2020 ([BEAM-8371]).

Reply:
Done